pulp_rpm/app/serializers/repository.py

```diff
@@ -124,6 +125,21 @@ class RpmPublicationSerializer(PublicationSerializer):
         help_text=_("An option specifying whether Pulp should generate SQLite metadata."),
     )
 
+    def validate(self, data):
+        """Validate data."""
+        if (
+            data["metadata_checksum_type"] not in settings.ALLOWED_CONTENT_CHECKSUMS
+            or data["package_checksum_type"] not in settings.ALLOWED_CONTENT_CHECKSUMS
+        ):
+            raise serializers.ValidationError(
+                _(
+                    "Checksum must be one of allowed types: {}."
+                    "You can adjust these with 'ALLOWED_CONTENT_CHECKSUMS' setting."
```

Comment:
@pavelpicka, `with the 'ALLOWED_CONTENT_CHECKSUMS' setting.` Don't forget `the`.
Please adjust in other places, I see it changed only here.
Not to adjust in many places, follow @dralley 's suggestion :D to put a part of the message into a variable.